sklego/model_selection.py

```diff
@@ -90,21 +88,33 @@ def __init__(
         self,
         date_serie,
         valid_duration,
+        stride_duration=None,
         train_duration=None,
         gap_duration=timedelta(0),
         n_splits=None,
         window="rolling",
     ):
+        # If stride length is not defined, set it equal to the length validation set
+        if stride_duration is None:
+            stride_duration = valid_duration
+
+        if stride_duration == timedelta(0):
+            raise ValueError("stride_duration should be longer than 0")
```

Comment:
Let's check for negative values entirely:

```suggestion
        if stride_duration <= timedelta(milliseconds=0):
            msg = f"`stride_duration` should be a positive timedelta, found {stride_duration}"
            raise ValueError(msg)
```